src/System.Net.Security/src/System/Net/HelperAsyncResults.cs

```diff
@@ -115,24 +132,18 @@ public bool MustCompleteSynchronously
         //
         // Important: This will abandon _Callback and directly notify UserAsyncResult.
         //
-        internal void CompleteWithError(Exception e)
-        {
-            UserAsyncResult.InvokeCallback(e);
-        }
+        internal void CompleteUserWithError(Exception e) => UserAsyncResult.InvokeCallback(e);
 
-        internal void CompleteUser()
-        {
-            UserAsyncResult.InvokeCallback();
-        }
+        internal void CompleteUser() => UserAsyncResult.InvokeCallback();
 
-        internal void CompleteUser(object userResult)
+        internal void CompleteUser(int userResult)
         {
```

Comment:
Oh, in the debug message below?  GitHub wasn't showing that on the comment view.  Will lower-case the R.